src/languages/es.ts

```diff
@@ -2616,4 +2616,13 @@ export default {
         taxRateChanged: 'La tasa de impuesto fue modificada',
         taxRequired: 'Falta tasa de impuesto',
     },
+    videoPlayer: {
+        play: 'Reproducir',
+        pause: 'Pausa',
+        fullscreen: 'Pantalla completa',
+        playbackSpeed: 'Velocidad de reproducción',
+        expand: 'Expandir',
+        mute: 'Silenciar',
+        unmute: 'Activar sonido',
```

Comment:
(Self-reminder to double-check on these translations with the team if no one has yet)